torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1564,20 +1588,20 @@ def sample_inputs_take_along_dim(op_info, device, dtype, requires_grad, **kwargs
             )
 
 def sample_inputs_amax_amin(op_info, device, dtype, requires_grad, **kwargs):
-    test_cases = (
-        ((S, S, S), ()),
-        ((S, S, S), (1,)),
-        ((S, S, S), ((1, 2,),)),
-        ((S, S, S), (1, True,)),
-        ((), (0,)),
-        ((), ()),
-        ((), (0, True,)),
+    test_cases: Tuple[tuple, tuple, dict] = (  # type: ignore[assignment]
```

Comment:
Document what each of these tuples is. Shape, positional args, kwargs?

Reply:
Done, fixed in the recent commit. Thanks!